Objects/longobject.c

```diff
@@ -3611,24 +3611,30 @@ long_richcompare(PyObject *self, PyObject *other, int op)
     Py_RETURN_RICHCOMPARE(result, 0, op);
 }
 
+static inline int
+/// Return 1 if the object is one of the immortal small ints
+_long_is_small_int(PyObject *op)
+{
+    PyLongObject *long_object = (PyLongObject *)op;
+
+    return (int)(long_object->long_value.lv_tag & IMMORTALITY_BIT_MASK);
```

Comment:
```suggestion
    return (long_object->long_value.lv_tag & IMMORTALITY_BIT_MASK) != 0;
```

This is exclusively used in tests.